packages/web-react/src/components/FileUploader/useFileUploaderAttachment.ts

```diff
@@ -1,4 +1,5 @@
-import { RefObject, useLayoutEffect } from 'react';
+import { RefObject } from 'react';
+import { useIsomorphicLayoutEffect } from '../../hooks/useIsomorphicLayoutEffect';
```

Comment:
```suggestion
import { useIsomorphicLayoutEffect } from '../../hooks';
```

And add useIsomorphicLayoutEffect export to the index.ts in the hooks directory, please.

Reply:
Ok, fixed.